sycl/tools/CMakeLists.txt

```diff
@@ -5,45 +5,29 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 add_executable(get_device_count_by_type get_device_count_by_type.cpp)
 add_dependencies(get_device_count_by_type ocl-headers ocl-icd)
 
-if( SYCL_BUILD_PI_CUDA )
```

Comment:
I removed this to avoid duplication and instead use the `cudadrv` target set up in the `sycl/plugin/cuda/CMakeLists.txt` CMake file.